application/build.gradle

```diff
@@ -1,12 +1,32 @@
 plugins {
-    id 'org.springframework.boot' version '2.7.8'
+	id 'org.springframework.boot' version '2.7.8'
+	id 'com.google.cloud.tools.jib' version '3.3.1'
 }
 
 dependencies {
-    implementation 'org.springframework.boot:spring-boot-starter-web'
-    implementation project(':infrastructure-petstore-rest-client')
-    implementation project(':domain')
+	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.springframework.boot:spring-boot-starter-actuator'
+	implementation project(':infrastructure-petstore-rest-client')
+	implementation project(':infrastructure-pet-jms')
+	implementation project(':domain')
 
-    developmentOnly 'org.springframework.boot:spring-boot-devtools'
-    testImplementation 'org.springframework.boot:spring-boot-starter-test'
-}
\ No newline at end of file
+	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	testImplementation 'org.awaitility:awaitility:4.2.0'
+	testImplementation 'org.springframework.cloud:spring-cloud-contract-wiremock'
+	testImplementation "org.junit.jupiter:junit-jupiter:5.8.1"
```

Comment:
you need a version here?

Reply:
I dont need the dep at all,  i would guess :D